targets/mbedos5/README.md

```diff
@@ -44,52 +44,32 @@ instructions above.
 make is used to automate the process of fetching dependencies, and making sure that
 mbed-cli is called with the correct arguments.
 
-### (optional) jshint
+### nodejs
 
-jshint is used to statically check your JavaScript code, as part of the build process.
-This ensures that pins you are using in your code are available on your chosen target
-platform.
+a recent version of [node.js](http://nodejs.org/).
```

Comment:
- Start sentence with capital letter. (Other sections start with lower case letters because they start with program names that cannot be capitalized.)
- What counts as recent? Is there a known working version or a minimum requirement?